float64 front_arms_sp
float64 rear_arms_sp 
